addons/core/translations/resources/en-us.yaml

```diff
@@ -464,6 +464,27 @@ credential-library:
     http_request_body:
       label: HTTP Method POST Request Body
       help: The body of the HTTP request the library sends to Vault when requesting credentials. Only allowed when HTTP method is set to POST.
+    username:
+      label: Username
+      help: This is the username that will be used to connect to Vault.
```

Comment:
I would change this to be inline with the rfc: `The username to use when making an SSH connection. This will be included in the valid_principles when making the request to Vault.`